arch/arm/src/cxd56xx/cxd56_allocateheap.c

```diff
@@ -115,7 +115,7 @@ void up_allocate_heap(void **heap_start, size_t *heap_size)
 
   board_autoled_on(LED_HEAPALLOCATE);
   *heap_start = (void *)g_idle_topstack;
-  *heap_size = (uint32_t)&__stack - g_idle_topstack;
+  *heap_size = (uint32_t)__stack - g_idle_topstack;
```

Comment:
```suggestion
  *heap_size = __stack - g_idle_topstack;
```

Reply:
Can't, since the type is different(uint8_t * v.s. uintptr_t).